[2022-05-23 20:15:56,063] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task scheduled__2022-02-08T00:00:00+00:00 [queued]>
[2022-05-23 20:15:56,080] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task scheduled__2022-02-08T00:00:00+00:00 [queued]>
[2022-05-23 20:15:56,082] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:56,083] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:15:56,083] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:56,125] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): thrid_task> on 2022-02-08 00:00:00+00:00
[2022-05-23 20:15:56,134] {standard_task_runner.py:52} INFO - Started process 2514 to run task
[2022-05-23 20:15:56,149] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', 'scheduled__2022-02-08T00:00:00+00:00', '--job-id', '589', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpfg5y68q7', '--error-file', '/tmp/tmp92th3upd']
[2022-05-23 20:15:56,151] {standard_task_runner.py:80} INFO - Job 589: Subtask thrid_task
[2022-05-23 20:15:56,341] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task scheduled__2022-02-08T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:15:56,660] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-08T00:00:00+00:00
[2022-05-23 20:15:56,661] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:15:56,662] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task3 and will be running after task1 at the same time as task2!']
[2022-05-23 20:15:56,689] {subprocess.py:85} INFO - Output:
[2022-05-23 20:15:56,702] {subprocess.py:92} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2022-05-23 20:15:56,712] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:15:56,795] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20220208T000000, start_date=20220523T201556, end_date=20220523T201556
[2022-05-23 20:15:56,883] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:15:56,982] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
